website/pages/docs/commands/operator/debug.mdx

```diff
@@ -0,0 +1,120 @@
+---
+layout: docs
+page_title: 'Commands: debug'
```

Comment:
```suggestion
page_title: 'Commands: operator debug'
```

Reply:
going off of what keyring had https://www.nomadproject.io/docs/commands/operator/keyring